web/ui/react-app/src/ExpressionInput.tsx

```diff
@@ -179,7 +191,10 @@ class ExpressionInput extends Component<ExpressionInputProps, ExpressionInputSta
                 </Button>
               </InputGroupAddon>
             </InputGroup>
-            {downshift.isOpen && this.renderAutosuggest(downshift)}
+            {downshift.isOpen &&
+              <div {...downshift.getMenuProps()} className="autosuggest-dropdown">
+                { this.creatAutocompleteSection(downshift) }
```

Comment:
Not sure what causes this, but with this PR there's a new bug where the dropdown opens very briefly and then closes immediately on any new input, even if the current string doesn't match anything in the history. E.g. type `zzzzzzzzz` and you'll see the (empty) dropdown briefly flash open after every typed letter.

Reply:
here is the fix https://github.com/prometheus/prometheus/pull/6211/commits/c9d2220113ef16d1d48b64387da78ae9f488d587